Samples/Samples/ViewModel/HomeViewModel.cs

```diff
@@ -156,6 +156,12 @@ public HomeViewModel()
                     typeof(PreferencesPage),
                     "Quickly and easily add persistent preferences.",
                     new[] { "settings", "preferences", "prefs", "storage" }),
+                new SampleItem(
+                    "⚙️",
```

Comment:
Maybe use the camera icon as this looks more settings-y.

Reply:
Where do you get the real icons from? I always just copied an existing icon as I don't know how to insert new icons. Is it as simple as opening charmap and copying any font icon in there?

Reply:
I just use sites like these: https://getemoji.com/

📷